modules/core/04-channel/keeper/timeout.go

```diff
@@ -208,7 +208,7 @@ func (k Keeper) TimeoutOnClose(
 	chanCap *capabilitytypes.Capability,
 	packet exported.PacketI,
 	proof,
-	proofClosed []byte,
+	closedProof []byte,
```

Comment:
`channelProof` would also be acceptable here I think, happy to keep with `closedProof` tho

Reply:
I think I could also rename `initProof`, `tryProof`, etc to `channelProof`, since that's what we're proving in the handshake steps, right?

Reply:
I will leave it as is and we can do more renaming later on.

Reply:
Sounds good!